### DISCUSSION TITLE
Add list of dependencies required to build awesome in README

### DISCUSSION THREAD

**Discussion by Blu3cr0ss** [awesomeWM/awesome#3916] [Ideas]
The problem is that when I want to build awesome I have to spend time finding and installing dependencies. I made a list of required and optional dependencies: 
```
# required
gcc
lua
git
cmake
imagemagick
pkgconf
# optional
asciidoctor
ldoc
```
I think it would be a good idea to add a command to install these dependencies in the README in the building and installation section. 
For example:
### Arch Linux
`pacman -S gcc lua git cmake imagemagick pkgconf`

etc...

**Comment by actionless** [comment]
it's there: https://awesomewm.org/apidoc/documentation/10-building-and-testing.md.html
